biogeochem/FatesAllometryMod.F90

```diff
@@ -407,15 +413,22 @@ subroutine bagw_allom(d,ipft,crowndamage, bagw,dbagwdd)
          call endrun(msg=errMsg(sourcefile, __LINE__))
       end select
       
+      ! Potentially reduce AGB based on crown damage (crown_reduction) and/or 
+      ! phenology (elongf_stem).
       if(crowndamage > 1) then
          call GetCrownReduction(crowndamage, crown_reduction)
-         bagw = bagw - (bagw * branch_frac * crown_reduction)
+         bagw = elongf_stem * ( bagw - (bagw * branch_frac * crown_reduction) )
```

Comment:
Yes, currently only non-woody plants can shed "stems". Indeed I decided to add the term there just in case we ever think woody plants should lose stems from drought, but currently elongf_stem should be always 1 for woody plants.